README.md

```diff
@@ -6,8 +6,8 @@
 </p>
 
 <p align="center">
-    <a href="https://github.com/alteryx/featuretools/actions?query=branch%3Amain+workflow%3ATests" target="_blank">
-        <img src="https://github.com/alteryx/featuretools/workflows/Tests/badge.svg?branch=main" alt="Tests" />
+    <a href="https://github.com/alteryx/featuretools/actions/workflows/tests_with_latest_deps.yaml" alt="Tests" target="_blank">
+        <img src="https://github.com/alteryx/featuretools/actions/workflows/tests_with_latest_deps.yaml/badge.svg?branch=main" alt="Tests" />
```

Comment:
When I follow this image link, it still shows "no status" for me. Should this be working? 

![image](https://github.com/alteryx/featuretools/assets/4307001/64f376a3-9fcc-4350-bf5f-e4e52af2ecbd)

Reply:
No, not yet